vendor-bin/psalm/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "config": {
+	"platform": {
+	    "php": "8.1"
+	},
+	"sort-packages": true
+    },
+    "require-dev": {
+        "vimeo/psalm": "^5.26"
```

Comment:
```suggestion
        "vimeo/psalm": "^6"
```

Don't know the current 6, please log it more specifically

Reply:
Sheesh 
![Screenshot_20250422_213955](https://github.com/user-attachments/assets/2a5c3d7c-a477-48de-be65-e05a9c7a4521)